dpsim/include/dpsim/MNASolverFactory.h

```diff
@@ -72,41 +63,71 @@ class MnaSolverFactory {
 	static std::shared_ptr<MnaSolver<VarType>> factory(String name,
 		CPS::Domain domain = CPS::Domain::DP,
 		CPS::Logger::Level logLevel = CPS::Logger::Level::info,
-		MnaSolverImpl implementation = mSupportedSolverImpls().back(),
+		DirectLinearSolverImpl implementation = DirectLinearSolverImpl::SparseLU,
 		String pluginName = "plugin.so")
 	{
 		//To avoid regression we use EigenDense in case of undefined implementation
```

Comment:
Comment still needs update to EigenSparse

Reply:
done